packages/puppeteer-extra-plugin-adblocker/src/index.ts

```diff
@@ -38,7 +40,8 @@ export class PuppeteerExtraPluginAdblocker extends PuppeteerExtraPlugin {
     return {
       blockTrackers: false,
       useCache: true,
-      cacheDir: undefined
+      cacheDir: undefined,
+      interceptResolutionPriority: 0
```

Comment:
I recommend defaulting to `undefined` to preserve legacy mode behavior. A priority should be explicitly set by the user if they want to use cooperative mode - at least, for now. Otherwise users may experience the handlers resolving in a different order than before, and that can be confusing.

tldr; Default to legacy mode and add a note to the README about how to enable cooperative mode for users who want it.

Reply:
Good point! I'll change it to be disabled by default 👍🏻